.taskcluster.yml

```diff
@@ -287,7 +287,117 @@ tasks:
-      description: Balrog Agent Docker Image Creation (release event)
+      name: Balrog Agent Docker Image Creation (release event) (py2)
+      description: Balrog Agent Docker Image Creation (release event) (py2)
+      owner: "{{ event.head.user.email }}"
+      source: "{{ event.head.repo.url }}"
+
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - secrets:get:repo:github.com/mozilla/balrog:dockerhub
+    payload:
+      maxRunTime: 3600
+      image: "taskcluster/image_builder:0.1.3"
+      features:
+        dind: true
+        taskclusterProxy: true
+      command:
+        - "/bin/bash"
+        - "-c"
+        - "git clone $GITHUB_HEAD_REPO_URL balrog && cd balrog && git checkout $GITHUB_HEAD_BRANCH && scripts/push-dockerimage.sh Dockerfile.py3 latest-py3 ${GITHUB_HEAD_BRANCH}-${GITHUB_HEAD_SHA}-py3"
+    extra:
+      github:
+        env: true
+        events:
+          - push
+        branches:
+          - master
+    metadata:
+      name: Balrog Docker Image Creation (py3)
+      description: Balrog Docker Image Creation (py3)
+      owner: "{{ event.head.user.email }}"
+      source: "{{ event.head.repo.url }}"
+
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - secrets:get:repo:github.com/mozilla/balrog:dockerhub
+    payload:
+      maxRunTime: 3600
+      image: "taskcluster/image_builder:0.1.3"
+      features:
+        dind: true
+        taskclusterProxy: true
+      command:
+        - "/bin/bash"
+        - "-c"
+        - "git clone $GITHUB_HEAD_REPO_URL balrog && cd balrog/agent && git checkout $GITHUB_HEAD_BRANCH && scripts/push-dockerimage.sh Dockerfile.py3 latest-py3 ${GITHUB_HEAD_BRANCH}-${GITHUB_HEAD_SHA}-py3"
+    extra:
+      github:
+        env: true
+        events:
+          - push
+        branches:
+          - master
+    metadata:
+      name: Balrog Agent Docker Image Creation (py3)
+      description: Balrog Agent Docker Image Creation (py3)
+      owner: "{{ event.head.user.email }}"
+      source: "{{ event.head.repo.url }}"
+
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - secrets:get:repo:github.com/mozilla/balrog:dockerhub
+    payload:
+      maxRunTime: 3600
+      image: "taskcluster/image_builder:0.1.3"
+      features:
+        dind: true
+        taskclusterProxy: true
+      command:
+        - "/bin/bash"
+        - "-c"
+        - "git clone -b {{ event.version }} {{ event.head.repo.url }} balrog && cd balrog && scripts/push-dockerimage.sh Dockerfile.py3 {{ event.version }}-py3"
+    extra:
+      github:
+        env: true
+        events:
+          - release
+    routes:
+      - index.project.balrog.releases.latest.app
+      - index.project.balrog.releases.{{ event.version }}.app
```

Comment:
Looks like these will stomp on the routes for the py2 release event. I'm not sure that will cause any problems but lets pre-emptively dodge.